src/variables.json

```diff
@@ -55,11 +55,11 @@
   "purple-lighter": "#E7E0F1",
   "purple-faint": "#F3F1F6",
 
-  "darken5": "rgba(0, 0, 0, 0.05)",
-  "darken10": "rgba(0, 0, 0, 0.1)",
-  "darken25": "rgba(0, 0, 0, 0.25)",
-  "darken50": "rgba(0, 0, 0, 0.5)",
-  "darken75": "rgba(0, 0, 0, 0.75)",
+  "darken5": "rgba(26, 34, 36, 0.05)",
+  "darken10": "rgba(26, 34, 36, 0.1)",
+  "darken25": "rgba(26, 34, 36, 0.25)",
+  "darken50": "rgba(26, 34, 36, 0.5)",
+  "darken75": "rgba(26, 34, 36, 0.75)",
```

Comment:
the darken classes are all now based on gray-dark.

Reply:
🤔 should we rgb everywhere instead of hex? annoying to translate between the two.

Reply:
I kind of wish we would change all of these to HSL instead. It's so much more readable